index.html

```diff
@@ -1,7 +1,97 @@
 ---
-layout: page
-title: cKeys
-subtitle: Mechanical Keyboard Workshops
+layout: landing
 ---
-<h2>Coming soon...</a></h2>
-  <p>Just another landing page for mechanical keyboard workshops in Seattle, Washington.</p>
+<!-- Banner -->
+<section id="banner">
+  <div class="inner">
+    <h2>{{ site.title }}</h2>
+    <p>{{ site.description | markdownify }}</p>
+    <ul class="actions">
+      <li><a href="/contact" class="button special">Contact</a></li>
+    </ul>
+  </div>
+  <a href="#one" class="more scrolly">Learn More</a>
+</section>
+
+<!-- One -->
+<section id="one" class="wrapper style1 special">
+  <div class="inner">
+    <header class="major">
+      <h2>We strive to make people smile <br/>one keyboard at a time.</h2>
+      <p>Meet local keyboard nerds at keyboard meetups. Learn to solder and program keyboards in our workshops. <br/>Help us nurture a positive and inclusive community for all ages.</p>
+    </header>
+    <ul class="icons major">
+      <li><span class="icon fa-keyboard-o major style2"><span class="label">Ipsum</span></span></li>
+      <li><span class="icon fa-code major style3"><span class="label">Dolor</span></span></li>
+      <li><span class="icon fa-laptop major style1"><span class="label">Lorem</span></span></li>
+    </ul>
+  </div>
+</section>
+
+<!-- Two -->
+<section id="two" class="wrapper alt style2">
+  <section class="spotlight">
+    <div class="image"><img src="/images/meetup-sign.jpg" alt="" /></div><div class="content">
+      <h2>We bring keyboard <br/>people together.</h2>
+      <p>And we bring people together through keyboards. We want you to join us at our next meetup or workshop whether it will be your first time touching a mechanical keyboard or you've been doing the mechanical since before it was hip.</p>
```

Comment:
"been doing the mechanical"?

I'm sure there is a better way to phrase that, but I'm not really sure how.